storage/innobase/handler/ha_innodb.cc

```diff
@@ -2719,6 +2719,18 @@ bool Encryption::is_keyring(const char *algoritm) {
   return (algoritm != NULL && innobase_strcasecmp(algoritm, "keyring") == 0);
 }
 
+bool Encryption::is_online_encryption_on() {
+  return srv_encrypt_tables == SRV_ENCRYPT_TABLES_ONLINE_TO_KEYRING ||
+         srv_encrypt_tables == SRV_ENCRYPT_TABLES_ONLINE_TO_KEYRING_FORCE;
+}
+
+// This for now excludes MK encryption ...
```

Comment:
Redundant comment given the method name?

Reply:
removed